game.js

```diff
@@ -1,6 +1,7 @@
 const GAME_SIZE = [50, 50];
 
 function gameInit() {
+    new GTheme().apply();
```

Comment:
Tohle bych řešil spíš v gamesStorage, není potreba to mít v kazde hre

Reply:
Jde o to, jestli to chceš v úplně každé hře aplikovat. Dokážu si představit situace, kdy nechceš měnit mezi dark a light modem.

Reply:
Tak pokud nechceš měnit, tak si napíšeš natvrdo barvy/přepíšeš proměnný, ne?
Mě to přijde jako fajn, mít zapnutý téma defaultně už v každý aplikaci

Reply:
Asi jo, jakože jenom nadefinovaný proměnný ti asi v ničem nevadí. Já jsem přemýšlel spíš do budoucna, kdyby jsme tu knihovnu nějak upravovali a ona dělala i nějaký jiný věci navíc, který by toho mohli měnit víc než jen názvy proměnných. Ale je fakt, že teď mně z halvy nic takovýho nenapadá a že kdyžtak tyhle věci můžeme případně zapínat ručně zvlášť. Takže ok, asi bude lepší to zapnout automaticky v gamesStorage.

Každopádně ten řádek kódu bych tam stejně nechal, jelikož když tam je, tak pokud si během hry uživatel změní motiv, pak  se změna aplikuje s každým zavoláním funkce init, tedy z každou novou hrou, kdežto kdyby tam nebyl, pak je potřeba refreshovat celou stránku nebo kompletně vypínat/zapínat celou hru, což není úplně ideální.

Reply:
Teď jsem to rozšířil tak, že se změna aplikuje okamžitě po zavolání apply a není potřeba dělat refresh 

https://github.com/QRGameStudio/web-qrpr.eu/blob/master/lib/themeProvider.js

Reply:
Ok, super, takže mažu a uvidíme, jak se to bude chovat.